doc_user/UserManual.adoc

```diff
@@ -3188,23 +3195,26 @@ Once the *[.underline]##U##sers* button in the *_Users & Groups_* submenu is pre
 
 image:image195.png[User Browser,width=621,height=305]
 
-The last column, *LOCKED*, if checked indicates that the user is currently locked out from logging into the system.
+The column, *LOCKED*, if checked indicates that the user is currently locked out from logging into the system.
+
+The last column, *DISABLED*, if checked indicates that the user has been marked as deleted(in that case, it cannot login into the application).
```

Comment:
```suggestion
The last column, *DELETED*, if checked indicates that the user has been marked as deleted and cannot be assigned to new users.
```

Reply:
Sorry, my bad, I thought we were talking about usergroups. Anyway the term you also used is "Deleted" (nod "Disabled") and the phrase looks like too similar to "Locked" definition. What about this?

```suggestion
The last column, *DELETED*, if checked indicates that the user has been marked as deleted and, unlike the locked user,  it will not be found at all at the login time.
```